packages/yswaps/contracts/TradeFactory/TradeFactoryPositionsHandler.sol

```diff
@@ -13,45 +13,40 @@ interface ITradeFactoryPositionsHandler {
     address _tokenOut;
   }
 
-  event TradeCreated(address indexed _strategy, address indexed _tokenIn, address indexed _tokenOut);
+  event TradeEnabled(address indexed _strategy, address indexed _tokenIn, address indexed _tokenOut);
```

Comment:
Since we are not longer really creating anything ... I thought "enable and disable" nomenclature was better suited.